src/pages/Favorites.js

```diff
@@ -46,3 +77,45 @@ const CardContainer = styled.div`
 const TextStyled = styled.h1`
   text-align: center;
 `
+
+const CardStyled = styled.section`
+  position: relative;
+  display: grid;
+  gap: 8px;
+  background-image: linear-gradient(60deg, #faf5ef, white);
+  padding: 22px 20px 16px;
+  border-radius: 15px;
+  box-shadow: 0 10px 10px #0002;
```

Comment:
Hex is a 6-digit number. If you need a color with 6 equal numbers, you can use 3 (#000, #FFF).